### ISSUE TITLE
Add to homebrew-cask-fonts

### ISSUE THREAD

**Issue by joseph-lozano** [vercel/geist-font#10]
It would be great if this could be added to https://github.com/Homebrew/homebrew-cask-fonts for easy installation for those that use Homebrew.

**Comment by JohnPhamous** [comment]
Want to make a PR?

**Comment by LeviticusNelson** [comment]
Added a PR: https://github.com/Homebrew/homebrew-cask-fonts/pull/8552

**Comment by joseph-lozano** [comment]
Looks it got merged!

**Comment by JohnPhamous** [comment]
Thank you @LeviticusNelson!

**Comment by AugustMiller** [comment]
Hey @JohnPhamous! @kennyheard and I noticed an issue when installing via Homebrew that may be the result of the [1.0.0](https://github.com/vercel/geist-font/releases/tag/1.0.0) release assets being updated, after-the-fact…

Kenny has submitted https://github.com/Homebrew/homebrew-cask-fonts/pull/8614 with the new checksums.

Unfortunately, it seems like release assets are _always_ mutable on GitHub—so a "soft" policy for maintainers is the only way to avoid this in the future. Figured it would be good to have you in the loop!
